src/Reflection/ReflectionProperty.php

```diff
@@ -317,7 +317,7 @@ public function __clone()
     /**
      * @param object|null $object
      *
-     * @return mixed
+     * @return Closure(object, string): mixed
```

Comment:
This isn't returning a callable

Reply:
Indeed. Must have overlooked __invoke call at the end.